client/stories/button.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Button from '../src/components/common/button/button.component';
+
+const title = 'SEARCH';
+const loadMoreTitle = 'Load More';
+const onClickSearchByButton = () => {
+  console.log('onClickSearchByButton');
+};
+storiesOf('Buttons', module)
```

Comment:
To follow  React's component principle, it's common to keep .stories files in the appropriate component's folder alongside the actual source .js file.
This way you'll have potentially more reusable components architecture that is working as a plugable piece with a default storybook config.